### PR TITLE
use feathers-action, rx, fp, ws

### PR DESCRIPTION
**Pull request by ahdinosaur**

### CODE REVIEW

**Comment by ahdinosaur** [comment]
ready!

dogstack example

GIF made using https://github.com/phw/peek, wow! :+1:

**Comment by ahdinosaur** [comment]
not sure why there's the flicker, also notice that adding a new dog gets slower with more dogs. keen to do easy optimizations for `feathers-action`, namely batch `set` and `unset`, since at the moment there are lots of these on every change.

**Comment by ahdinosaur** [comment]
okay fixed the flicker: https://github.com/ahdinosaur/feathers-action/pull/13

dogstack1

File: store.js [hunk 1]
```diff
@@ -1,24 +1,32 @@
...
 
-import rootReducer from './reducer'
-import client from './client'
+import feathers from './client'
+import rootUpdater from './updater'
+import rootEpic from './epic'
 
 const middleware = [
-  thunk.withExtraArgument(client),
+  thunk.withExtraArgument(feathers),
```
Comment (pietgeursen) [inline store.js]: think this can be removed can't it?

**Review by pietgeursen — APPROVED** [review]
Oooooooh yeah! :tada: :1st_place_medal: :poodle:

File: store.js (hunk 1 above)
Comment (ahdinosaur) [reply store.js]: it's still used by authentication, yeah? [accounts/actions.js](https://github.com/dogstack/dogstack-example/blob/feathers-action/account/actions.js)

**Pull request merged**